pkg/capacity/pci_test.go

```diff
@@ -0,0 +1,34 @@
+package capacity
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestGetDevice(t *testing.T) {
+	vendor, device, ok := GetDevice(0x1002, 0x731f)
+
+	require.True(t, ok)
+	require.Equal(t, "Advanced Micro Devices, Inc. [AMD/ATI]", vendor.Name)
+	require.Equal(t, "Navi 10 [Radeon RX 5600 OEM/5600 XT / 5700/5700 XT]", device.Name)
+}
+
+func TestListPCI(t *testing.T) {
+	devices, err := ListPCI()
+	require.NoError(t, err)
+
+	for _, device := range devices {
+		fmt.Println(device)
```

Comment:
Shouldn't we assert here for inclusion of a device in the devices slice?

Reply:
Well, the returned values are not predictable because they completely depends on the machine they run (the actual PCI in the systems, or the GPUs if exists. The prints where mainly for during debugging but there are no solid values i can assert against